node/heartbeat/sender.go

```diff
@@ -36,16 +36,22 @@ func NewSender(
 	if peerMessenger == nil {
 		return nil, ErrNilMessenger
 	}
+
 	if singleSigner == nil {
 		return nil, ErrNilSingleSigner
 	}
+
 	if privKey == nil {
 		return nil, ErrNilPrivateKey
 	}
+
```

Comment:
leave as it was before. without empty lines between nil checks

Reply:
done